mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantsFragment.java

```diff
@@ -91,7 +92,16 @@ private void setupRecyclerView() {
                 new RecyclerItemClickListener.OnItemClickListener() {
                     @Override
                     public void onItemClick(View childView, int position) {
-
+                        Log.e("Cool", "Not Soo Cool");
```

Comment:
@Vipinsoni38 Can you please remove the log message as it is not necessary.

Reply:
Done! :)

Reply:
Nice